core/src/main/java/org/stellar/anchor/sep6/Sep6Service.java

```diff
@@ -372,6 +373,7 @@ public StartWithdrawResponse withdrawExchange(
           exchangeAmountsCalculator.calculateFromQuote(
               request.getQuoteId(), sellAsset, request.getAmount());
     } else {
+      // TODO(philip): remove this
```

Comment:
Why not remove?

Reply:
I will need to update the end-to-end tests which will take some time, but I want to this PR merged into the 2.9 release. The SEP-6 changes can be released in 2.9.1.

Reply:
It would be nice to create a ticket for the end-to-end tests update. Otherwise, LGTM.

Reply:
Yep. I'll create the tickets. Thanks!